.gitignore

```diff
@@ -159,4 +159,4 @@ cython_debug/
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
-#.idea/
+.idea
```

Comment:
Other projects have also:
```suggestion
# idea
.idea
*.iml
logs

# vs code
.vscode
```

Reply:
Thanks. Address in commit 0b351127825f0dd446f0a0828295429ff5627f10.